package.json

```diff
@@ -1,47 +1,57 @@
 {
-  "name": "task-manager-app",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@mui/lab": "5.0.0-alpha.57",
-    "@reduxjs/toolkit": "1.6.2",
-    "@testing-library/jest-dom": "5.11.4",
-    "@testing-library/react": "11.1.0",
-    "@testing-library/user-event": "12.1.10",
-    "@types/jest": "26.0.15",
-    "@types/node": "12.0.0",
-    "@types/react": "17.0.0",
-    "@types/react-dom": "17.0.0",
-    "formik": "2.2.9",
-    "formik-mui-lab": "1.0.0-alpha.3",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-scripts": "4.0.3",
-    "typescript": "4.1.2",
-    "web-vitals": "1.0.1"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  }
+    "name": "task-manager-app",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@mui/lab": "5.0.0-alpha.57",
+        "@reduxjs/toolkit": "1.6.2",
+        "@testing-library/jest-dom": "5.11.4",
+        "@testing-library/react": "11.1.0",
+        "@testing-library/user-event": "12.1.10",
+        "@types/jest": "26.0.15",
+        "@types/node": "12.0.0",
+        "@types/react": "17.0.0",
+        "@types/react-dom": "17.0.0",
+        "formik": "2.2.9",
+        "formik-mui-lab": "1.0.0-alpha.3",
+        "prettier": "2.5.0",
+        "react": "17.0.2",
+        "react-dom": "17.0.2",
+        "react-scripts": "4.0.3",
+        "typescript": "4.1.2",
+        "web-vitals": "1.0.1"
+    },
+    "scripts": {
+        "start": "react-scripts start",
+        "build": "react-scripts build",
+        "test": "react-scripts test",
+        "eject": "react-scripts eject",
+        "prepare": "husky install"
+    },
+    "browserslist": {
+        "production": [
+            ">0.2%",
+            "not dead",
+            "not op_mini all"
+        ],
+        "development": [
+            "last 1 chrome version",
+            "last 1 firefox version",
+            "last 1 safari version"
+        ]
+    },
+    "devDependencies": {
+        "@typescript-eslint/eslint-plugin": "5.4.0",
+        "@typescript-eslint/parser": "5.4.0",
+        "eslint": "7.32.0",
+        "eslint-config-airbnb": "19.0.1",
+        "eslint-config-prettier": "8.3.0",
+        "eslint-import-resolver-typescript": "2.5.0",
+        "eslint-plugin-import": "2.25.3",
+        "eslint-plugin-jsx-a11y": "6.5.1",
+        "eslint-plugin-prettier": "4.0.0",
+        "eslint-plugin-react": "7.27.1",
+        "eslint-plugin-react-hooks": "4.3.0",
+        "husky": "^7.0.4"
```

Comment:
fixed version

Reply:
check eslint for rule to have fixed versions in packages.json

Reply:
I haven't found such rule